src/packages.config

```diff
@@ -24,12 +26,10 @@
   <package id="Microsoft.VisualStudio.Threading" version="14.1.131" targetFramework="net451" />
   <package id="Microsoft.VisualStudio.Utilities" version="14.2.25123" targetFramework="net451" />
   <package id="Microsoft.VisualStudio.Validation" version="14.1.111" targetFramework="net451" />
-  <package id="Microsoft.VSSDK.BuildTools" version="14.2.25201" targetFramework="net451" developmentDependency="true" />
+  <package id="Microsoft.VSSDK.BuildTools" version="15.0.26201" targetFramework="net451" developmentDependency="true" />
   <package id="TemplateBuilder" version="1.1.4.5-beta" targetFramework="net451" />
-  <package id="Cake.Core" version="0.17.0" targetFramework="net451" developmentDependency="true" />
-  <package id="Cake.Testing" version="0.17.1" targetFramework="net451" developmentDependency="true" />
   <package id="xunit" version="2.1.0" targetFramework="net451" developmentDependency="true" />
-  <package id="xunit.abstractions" version="2.0.1" targetFramework="net451" developmentDependency="true" />
+  <package id="xunit.abstractions" version="2.0.0" targetFramework="net451" developmentDependency="true" />
```

Comment:
@agc93 you should also upgrade the `<Content/>` element in [`Cake.VisualStudio.csproj`](https://github.com/agc93/cake-vs/blob/5531b091c7de232f9911a97602ef48f78eb9bb2b/src/Cake.VisualStudio.csproj#L169). Otherwise the build will not work. Probably your build worked because you did not cleaned the packages folder.